graylog2-server/src/main/java/org/graylog2/dashboards/widgets/FieldChartWidget.java

```diff
@@ -65,15 +66,11 @@ public FieldChartWidget(MetricRegistry metricRegistry, Searches searches, String
         final ImmutableMap.Builder<String, Object> persistedConfig = ImmutableMap.<String, Object>builder()
                 .put("query", query)
                 .put("timerange", timeRange.getPersistedConfig())
-                .put("field", config.get("field"))
-                .put("valuetype", config.get("valuetype"))
-                .put("renderer", config.get("renderer"))
-                .put("interpolation", config.get("interpolation"))
-                .put("interval", config.get("interval"));
-
-        if (!isNullOrEmpty(streamId)) {
-            persistedConfig.put("stream_id", streamId);
-        }
+                .put("field", field)
+                .put("valuetype", statisticalFunction)
+                .put("renderer", renderer)
+                .put("interpolation", interpolation)
+                .putAll(super.getPersistedConfig());
```

Comment:
Shouldn't this be the first addition to the map so that the query parameters and the local config can overwrite settings?